### PR TITLE
Clean nintegrate revised

### PR DESCRIPTION
**Pull request by raymond-sde**

### CODE REVIEW

File: src/com/infected/controller/Game.java [hunk 1]
```diff
@@ -56,8 +54,8 @@ public void start() {
         }
 
     }
-public boolean isSynonym(String input){
 
+public String getProcessedVerb(String input){
```
Comment (raymond-sde) [inline src/com/infected/controller/Game.java:58]: fix indentation

File: src/com/infected/controller/Game.java [hunk 2]
```diff
@@ -66,23 +64,24 @@ public boolean isSynonym(String input){
...
+                return "get";
             }
         }
     }catch (IOException e){
         e.printStackTrace();
     }
     System.out.println(input + " :not found");
-    return false;
+    return input;
 }
```
Comment (raymond-sde) [inline src/com/infected/controller/Game.java:86]: fix indentation

File: src/com/infected/controller/Game.java [hunk 3]
```diff
@@ -66,23 +64,24 @@ public boolean isSynonym(String input){
         for(int i =0; i <(node.get("commands").get("go").get("synonyms").size());i++){
```
Comment (raymond-sde) [inline src/com/infected/controller/Game.java:64]: redundant logic. only need 1 for loop for all verbs

**Review by GeniusAlg0s — COMMENTED** [review]
will make future changes to the file

**Pull request merged**